.github/actions/build/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Container image that runs your code
+FROM adoptopenjdk/openjdk8
+
+# install git so we can get version label
+RUN apt -y update;apt -y install git zip
+
+# Copies your code file from your action repository to the filesystem path `/` of the container
+COPY entrypoint.sh /entrypoint.sh
+
+# Code file to execute when the docker container starts up (`entrypoint.sh`)
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
Installing git where?

For the remaining ".github/..." entries, I will view them but I do not have enough git expertise to check if they are valid.

Reply:
This file sets up a docker container to run the build. The container starts with a Java 8 image and I then install gut for the new build functions. This is all GitHub Actions stuff if you want to learn more.